### PR TITLE
fix: use getDevice in deletes worker

### PR DESCRIPTION
**Pull request by christyjacob4**
## What does this PR do?

Use the `getDevice` function inside the deletes worker where we were still hardcoding the `Local` device.

## Test Plan
NA

## Related PRs and Issues
NA

### Have you added your change to the [Changelog](https://github.com/appwrite/appwrite/blob/master/CHANGES.md)?
YES

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

YES

### CODE REVIEW

**Review by Meldiron — APPROVED** [review]

**Review by TorstenDittmann — APPROVED** [review]

File: app/workers/deletes.php [hunk 1]
```diff
@@ -259,8 +259,8 @@ protected function deleteProject(Document $document): void
         $this->getProjectDB($projectId)->delete($projectId);
 
         // Delete all storage directories
-        $uploads = new Local(APP_STORAGE_UPLOADS . '/app-' . $document->getId());
-        $cache = new Local(APP_STORAGE_CACHE . '/app-' . $document->getId());
+        $uploads = $this->getDevice(APP_STORAGE_UPLOADS . '/app-' . $document->getId());
+        $cache = $this->getDevice(APP_STORAGE_CACHE . '/app-' . $document->getId());
```
Comment (stnguyen90) [inline app/workers/deletes.php:263]: Cache seems to always be local file system:

https://github.com/appwrite/appwrite/blob/02a432102df047d2ea96faed65d77a862239d487/app/controllers/shared/api.php#L166

File: app/workers/deletes.php [hunk 2]
```diff
@@ -425,7 +425,7 @@ protected function deleteFunction(Document $document, string $projectId): void
          * Delete Deployments
          */
         Console::info("Deleting deployments for function " . $functionId);
-        $storageFunctions = new Local(APP_STORAGE_FUNCTIONS . '/app-' . $projectId);
+        $storageFunctions = $this->getDevice(APP_STORAGE_FUNCTIONS . '/app-' . $projectId);
```
Comment (stnguyen90) [inline app/workers/deletes.php:428]: Looks like there should already be a [`getFunctionsDevice()`](https://github.com/appwrite/appwrite/blob/02a432102df047d2ea96faed65d77a862239d487/src/Appwrite/Resque/Worker.php#L254) method on the Worker. Maybe that can be reused?

File: app/workers/deletes.php [hunk 3]
```diff
@@ -259,8 +259,8 @@ protected function deleteProject(Document $document): void
         $this->getProjectDB($projectId)->delete($projectId);
 
         // Delete all storage directories
-        $uploads = new Local(APP_STORAGE_UPLOADS . '/app-' . $document->getId());
-        $cache = new Local(APP_STORAGE_CACHE . '/app-' . $document->getId());
+        $uploads = $this->getDevice(APP_STORAGE_UPLOADS . '/app-' . $document->getId());
```
Comment (stnguyen90) [inline app/workers/deletes.php:262]: Looks like there's already a [`getFIlesDevice()`](https://github.com/appwrite/appwrite/blob/02a432102df047d2ea96faed65d77a862239d487/src/Appwrite/Resque/Worker.php#L264) method on Worker. Maybe that can be reused?

File: app/workers/deletes.php [hunk 4]
```diff
@@ -442,7 +442,7 @@ protected function deleteFunction(Document $document, string $projectId): void
          * Delete builds
          */
         Console::info("Deleting builds for function " . $functionId);
-        $storageBuilds = new Local(APP_STORAGE_BUILDS . '/app-' . $projectId);
+        $storageBuilds = $this->getDevice(APP_STORAGE_BUILDS . '/app-' . $projectId);
```
Comment (stnguyen90) [inline app/workers/deletes.php:445]: Looks like there's already a [`getBuildsDevice()`](https://github.com/appwrite/appwrite/blob/02a432102df047d2ea96faed65d77a862239d487/src/Appwrite/Resque/Worker.php#L275) method on the Worker. Maybe that can be re-used?

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

File: app/workers/deletes.php (hunk 2 above)
Comment (christyjacob4) [reply app/workers/deletes.php:428]: Done

File: app/workers/deletes.php (hunk 3 above)
Comment (christyjacob4) [reply app/workers/deletes.php:262]: done

File: app/workers/deletes.php (hunk 4 above)
Comment (christyjacob4) [reply app/workers/deletes.php:445]: done

File: app/workers/deletes.php (hunk 1 above)
Comment (christyjacob4) [reply app/workers/deletes.php]: Fixed

**Review by stnguyen90 — APPROVED** [review]

**Review by Abhay-tech-boy — APPROVED** [review]
lgtm!

**Review by eldadfux — APPROVED** [review]

**Pull request merged**